backend/etc/device.xml

```diff
@@ -0,0 +1,26 @@
+<device_list>
```

Comment:
For now, this is ok but we should be getting it from raptor repo as discussed.

Reply:
noted.